scripts/ci/install_node_modules.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 
-set -x #echo on
+set -x # echo on
 
 cd bokehjs
 npm install --location=global npm
 npm ci --no-progress
```

Comment:
not sure if it is relevant any longer but this appears to be why the examples build was added originally:

https://github.com/bokeh/bokeh/pull/10058#issuecomment-633722464

Reply:
It is relevant for bokeh-ci and docker workflows. bokehjs-ci does this on its own.